requirements/edx/base.txt

```diff
@@ -110,14 +110,14 @@ edx-opaque-keys[django]==2.0.1  # via -r requirements/edx/paver.txt, edx-bulk-gr
 edx-organizations==4.0.0  # via -r requirements/edx/base.in
 edx-proctoring-proctortrack==1.0.5  # via -r requirements/edx/base.in
 edx-proctoring==2.3.0     # via -r requirements/edx/base.in, edx-proctoring-proctortrack
-edx-rbac==1.1.0           # via edx-enterprise
+edx-rbac==1.1.1           # via edx-enterprise
 edx-rest-api-client==4.0.1  # via -r requirements/edx/base.in, edx-enterprise, edx-proctoring
 edx-search==1.3.3         # via -r requirements/edx/base.in
 edx-sga==0.10.0           # via -r requirements/edx/base.in
 edx-submissions==3.0.4    # via -r requirements/edx/base.in, ora2
 edx-tincan-py35==0.0.5    # via edx-enterprise
 edx-user-state-client==1.1.2  # via -r requirements/edx/base.in
-edx-when==1.0.2           # via -r requirements/edx/base.in, edx-proctoring
+edx-when==1.0.3           # via -r requirements/edx/base.in, edx-proctoring
```

Comment:
This is going to bite us in the ass for a while, but I think this upgrade causes tests to fail.